src/components/button/config/index.js

```diff
@@ -13,91 +10,99 @@ export const args = {
   narrow: false,
 };
 
-export const argTypes = {
-  secondary: {
-    name: 'data-vl-secondary',
-    type: { summary: 'boolean' },
-    description: 'Attribuut wordt gebruikt in combinatie met een gewone knop om alternatieve acties te voorzien.',
-    table: {
-      defaultValue: { summary: 'false' },
-      category: CATEGORIES.ATTRIBUTES,
-    },
-  },
-  tertiary: {
-    name: 'data-vl-tertiary',
-    type: { summary: 'boolean' },
-    description:
-      'Attribuut wordt gebruikt in combinatie met gewone en secondary knoppen om alternatieve acties te voorzien.',
-    table: {
-      defaultValue: { summary: 'false' },
-      category: CATEGORIES.ATTRIBUTES,
-    },
-  },
+export const argsButton = {
```

Comment:
Het lijkt me beter om hier uw shared args in te spreaden tov in uw stories file. Dan kan je in uw stories file gewoon zeggen `args: buttonArgs`